modules/ocf/manifests/packages/dolphin_emu.pp

```diff
@@ -0,0 +1,3 @@
+class ocf::packages::dolphin_emu {
```

Comment:
If this package doesn't have a special setup needed to install it, you don't need to make a new Puppet class for it, just add it to the list of desktop packages at https://github.com/ocf/puppet/blob/master/modules/ocf_desktop/manifests/packages.pp